app/views/root/local_transaction.html.erb

```diff
@@ -21,10 +21,10 @@
        <span class="destination"> on <%= @interaction_details['local_authority']['name'] %></span>
     </p>
 
-  <% elsif @interaction_details['local_authority'] %>
+  <% elsif @location_error && @location_error.postcode_error == "laMatchNoLink" %>
```

Comment:
Minor nitpick: consider wrapping the 2nd part of this in a method on `LocationError`.  "laMatchNoLink" isn't super clear to the next developer, but `@location_error.no_location_interaction?` would be.